packages/nouns-webapp/src/components/TightStackedCircleNouns/index.tsx

```diff
@@ -0,0 +1,64 @@
+import { getNoun } from '../StandaloneNoun';
+import { BigNumber } from 'ethers';
+import { useNounSeeds } from '../../wrappers/nounToken';
+
+interface StackedCircleNounsProps {
+  nounIds: Array<number>;
+}
+
+// TODO
+const MAX_NOUNS_PER_STACK = 3;
+
+const TightStackedCircleNouns: React.FC<StackedCircleNounsProps> = props => {
+  const { nounIds } = props;
+
+  // TODO not sure if this will be an issue for new nouns or something?
+  const seeds = useNounSeeds();
```

Comment:
This will eventually get populated, but is not guaranteed to exist. Haven't seen that cause any issues yet, however it does seem possible that the `seeds[nounId]` lookup throws because `seeds` is undefined.

Reply:
Added some logic here to hopefully give us just a bit more insurance against this edge case